roles/pulp_common/vars/main.yml

```diff
@@ -51,3 +51,5 @@ __galaxy_lock_requirements: '{{ galaxy_lock_requirements | default(0) }}'
 
 # Pulps own replacement for django-admin setup in the proper environment
 pulp_django_admin_path: "{{ pulp_install_dir }}/bin/pulpcore-manager"
+
+pulp_content_path_prefix: "{{ pulp_settings.content_path_prefix | default('/pulp/content') }}"
```

Comment:
Can't we just access pulp_settings.content_path_prefix directly throughout the code?

That is basically my plan for the installer, and to document those special pulp_settings variables.

See in this vars file `__pulp_common_pulp_settings_defaults`

Reply:
Direct read of pulp_settings is not possible as we do not update it with the defaults (only when writing config file but not saving back to pulp_settings).

Reply:
Can't we access `__pulp_common_merged_pulp_settings.content_path_prefix` instead? I think that is the correct variable to access.